cdap-proto/src/main/java/co/cask/cdap/proto/ops/DashboardProgramRunRecord.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.ops;
+
+import co.cask.cdap.api.artifact.ArtifactId;
+import co.cask.cdap.api.artifact.ArtifactSummary;
+import co.cask.cdap.proto.ProgramRunStatus;
+import co.cask.cdap.proto.RunRecord;
+import co.cask.cdap.proto.id.ProgramRunId;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents a record of a program run information to be included in a dashboard detail view.
+ */
+public class DashboardProgramRunRecord {
```

Comment:
According to design, application should include name and version, its missing here, please fix
```
{
    "namespace":"<namespace>", 
    "artifact": {
      "scope" : "<scope>",
      "name" : "<name>",
      "version" : "<version>"
    },
    "application": {
      "name" : "<name>",
      "version" : "<version>"
    },
    "type":"<type>",
    "program":"<program>",
    "run" : "<run>",
    "user": "<user>",
    "startMethod" : "<manual, scheduled or triggered>",
    "start": "<start>",
    "running": "<running>",
    "end":"<end>",
    "status":"<status>"
  }
```